### ISSUE TITLE
OnPress function on Pie Chart?

### ISSUE THREAD

**Issue by htduongdl96** [Abhinandan-Kushwaha/react-native-gifted-charts#111]
Is Pie Chart have an onPress function, I don't see it on prop and documentation?

**Comment by Abhinandan-Kushwaha** [comment]
Hi @htduongdl96 👋

This is perhaps not available in the current version. I have added it to the [Todo list](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/src/todos.md). Will be adding this feature in the next release.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @htduongdl96 👋
Thanks for requesting the feature. `onPress` is now available in version `1.2.6`🎉

`onPress` prop takes a callback function with 2 parameters- item and index. Here's an example-
```js
onPress = { (item, index) => Alert.alert(item.value.toString() ) }
```

onPress can also be passed separately for each data item inside the data array.
 
---

Exactly similar to onPress, we have another prop named `onLabelPress`. As the name suggests, this function is called when a label is pressed.
